packages/component-library/src/Link.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import cx from 'clsx';
+import { processStyle, createStyleBuilder, createBootstrap, StyleConfig as BaseStyleConfig } from './helpers';
+
+export interface Props {
+  id?: string;
+  className?: string;
+  style?: object;
+  href?: string;
+  [key: string]: any;
+}
```

Comment:
Does it need a "content" prop or should the displayed content be passed as a child component?

Reply:
I went with child since other base components have that (nothing has a `content` prop).

Reply:
Along with the changes to the type of the `children` prop above, this looks good